test/unit/test-amp-story-player.js

```diff
@@ -209,43 +217,49 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
     buildStoryPlayer(2);
 
     await manager.loadPlayers();
+    await nextTick();
 
     const fakeData = {next: true};
     fireHandler['selectDocument']('selectDocument', fakeData);
 
-    win.requestAnimationFrame(() => {
-      const iframes = playerEl.shadowRoot.querySelectorAll('iframe');
-      expect(iframes[0].getAttribute('i-amphtml-iframe-position')).to.eql('-1');
-      expect(iframes[1].getAttribute('i-amphtml-iframe-position')).to.eql('0');
-    });
+    const iframes = playerEl.shadowRoot.querySelectorAll('iframe');
+    await afterRenderPromise();
+
+    // TODO(Enriqe): replace with navigation API once ready.
```

Comment:
Do you want this TODO to be on the `afterRenderPromise`? If you want a ticket plz add its # :))